exporters/otlp/src/otlp_metric_utils.cc

```diff
@@ -252,7 +254,9 @@ void OtlpMetricUtils::PopulateResourceMetrics(
     proto::common::v1::InstrumentationScope *scope = scope_lib_metrics->mutable_scope();
     scope->set_name(scope_metrics.scope_->GetName());
     scope->set_version(scope_metrics.scope_->GetVersion());
-    resource_metrics->set_schema_url(scope_metrics.scope_->GetSchemaURL());
```

Comment:
This looks like a bug. Setting the scope metrics schema at the resource metrics level.